xamarin/Xamarin.Forms
# [Android] Fix wrong SwipeView overlapping issue

### Description of Change ###

 Fix wrong SwipeView overlapping issue on Android. When reloading the list, the content of the SwipeView was being rendered keeping the previous one.

### Issues Resolved ### 

- fixes #12750 

### API Changes ###
 
 None

### Platforms Affected ### 

- Android

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 

iOS
![fix12750-ios](https://user-images.githubusercontent.com/6755973/98561383-2164be80-22a9-11eb-8a2d-eb297f7f4942.gif)
Android
![fix12750-droid](https://user-images.githubusercontent.com/6755973/98561359-19a51a00-22a9-11eb-8434-e9a018daebcd.gif)

### Testing Procedure ###
Launch Core Gallery and navigate to the issue 12750. Tap the Button, add a new Item and then, check open the first SwipView. Without duplicate items or a wrong layout, the test has passed.

### PR Checklist ###
